code-challenges.js

```diff
@@ -3,39 +3,102 @@
+        
+// // var middleLetters1 = "hello"
+// // // // // Expected output: “l”
+// // var middleLetters2 = "rhinoceros"
+// // // // // Expected output: “oc”
+
+// // var split = middleLetters1.split(" ");
+
+// // //splits word into individual strings in a new array
+// // //once word is in it's own array, .length indicates the length of the array 
+// // // 
+// // var middle = Math.floor(middleLetters1.length / 2);
+
+
+
+
+
+// // // --------------------4) Create a class to get the area of a sphere. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
+
+// class Sphere {
+//     constructor (sphere) {
+//         this.volume = 4 * Math.PI * radius * radius 
+// }
+//     string (){
+//         return `the radius of the sphere is  ${this.volume}`
+//     }
+// }
+
+// let sphere1 = this.volume = 4
+// let sphere 2 = this.volume = 2
+// let sphere 3 = this.volume = 6
+
+// // --------------------5) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+// function additUp() {
+//     let array = []
+//     for (let i=0;i<array.length;i++) {
+  
+//         return i++array.length
+//     }
+    
+// }
+
+// console.log(additUp(numbersToAdd1))
 
+var numbersToAdd1 = [2, 4, 45, 9]
+// // Excpected output: [2, 6, 51, 60]
+// var numbersToAdd2 = [0, 7, -8, 12]
+// // Expected output: [0, 7, -1, 11]
+// var numbersToAdd3 = []
+// // Expected output: []
+ 
 
+function additUp (array) {
 
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
-
-var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 9, 7, 199]
-var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
-
-
-
-
-// --------------------3) Create a function that takes in a string of a single word and returns the middle letter of the word. If the word is an even number of letters, return the two middle letters.
-
-var middleLetters1 = "hello"
-// Expected output: “l”
-var middleLetters2 = "rhinoceros"
-// Expected output: “oc”
-
-
-
-
-// --------------------4) Create a class to get the area of a sphere. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
-
-
+    var arrayTotal =array.length; 
+    var totalSum = 0;
 
+    for(var x = 0; x < arrayTotal; x++){
+        totalSum += array[x];
+    }
 
-// --------------------5) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+    return(totalSum); 
+}
 
-var numbersToAdd1 = [2, 4, 45, 9]
-// Excpected output: [2, 6, 51, 60]
-var numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
-var numbersToAdd3 = []
-// Expected output: []
+console.log (additUp (numbersToAdd1))
```

Comment:
make sure you comment your code back _in_ before you push up. it makes it much easier to read and test if it's not commented out.